conf/application.conf

```diff
@@ -204,6 +204,9 @@ object-store.default-retention-period = "1-week"
 internal-auth.token = "123457"
 
 internal-auth {
+   # used for talking to object-store
+    token = "123457"
```

Comment:
you can remove the duplicate `internal-auth.token` config just above this now

Reply:
whoops, missed that

Reply:
[updated](https://github.com/hmrc/hec/pull/55/commits/9358cc4ca0b449bac2ab68e9722ceed844017795)